lib/awful/spawn.lua

```diff
@@ -323,14 +323,12 @@ end
 --- Asynchronously spawn a program and capture its output.
 -- (wraps `spawn.with_line_callback`).
 -- @tparam string|table cmd The command.
--- @function callback Function with the following arguments:
+-- @tab callback Function with the following arguments
```

Comment:
`@tab` is apparently our own invention (`docs/config.ld` defines it) and it is just a short-hand for `@tparam table`.
`@function` is wrong since it documents a (global) function, for example `math.max`. It does not document parameters to a function.
(`@table` would also be wrong here, which is likely what you were thinking of: It documents a global table and not a table argument.)